Organisation switcher

gavinwye opened the pull request:
This PR adds the organisation switcher to all pages across the service.

This will need to be removed from the notifications page but that requires creating a new page template, so I'll do that shortly

gavinwye committed:
Added the organisation switcher

RoseFaulkner commented:
My recollection from UR is that some folk didn't notice this line at the top like this. Or am I misrememberIng?  
On the live version, the line 'Below are prisoners who are at **HMP Albany** has tested well I think.

gavinwye committed:
New layout for notifications page

This layout removes the organisation switcher from the top of the notifications page

gavinwye commented:
My recollection from the last round of UR was that all users understood why they were seeing this at the top of the page. 

I haven't seen enough of the research for the line _'Below are prisoners who are at HMP Albany'_ but based on my experience I think displaying a list of all of the variables that the data is filtered by at the top of the page would be missed. Normally in faceted search, you'd display these as discrete words that can be removed easily.

As this is a global setting it needs to stay at the top of the page because it impacts more than just search. That aim of this is to make it clear to the user that all of the data (with the exception of Notifications*) that they are seeing on the site is related to a particular location.

I thought this is what we discussed yesterday and everyone agreed to put this back in.

* I'm not completely convinced that's the right approach at the moment.

RoseFaulkner commented:
Hiya, yep, that all makes sense. Maybe I am misremembering about some users not noticing it in the last round? We did agree to put it back, it just wasn't until I saw it again that I remembered about the UR and whether we should make it more eye catching? You decide. 

Btw, your pt about not being convinced this is the right approach at the moment. I think it's important to include it but if you're concerned we should talk about this further. 

Also, can I check, is it only the organisation switcher we should be looking at right now? I noticed some other changes (based on our conversation in story time yesterday) but I didn't look at/comment on  these as I assumed you weren't ready for that yet?

janereid73 commented:
A couple of points from me: 
1. We conducted the research because there it was recognised that search and filter needed improved. A regular user in this round of testing, who uses the live search for 90% of what he does commented on how much cleaner it was in terms of design. All other users coming at it first time successfully completed the search task - it was the results section most had to re read before understanding what was being played back to them.
2. 'Selecting prisons' and 'search' are two different things. 'Selecting prisons' would fall into the category of 'settings' on a page /site. Not the search functionality.
3.  In the last round of UR users who didn't mention unprompted what prisons they were looking at were probed and told us what prisons they were looking at. It was the top navigation bar people didn't see or use to navigate through the pages not the prison selector.  Here's the link to the findings for ease if you wish to look back it. https://docs.google.com/presentation/d/1tq9KCvUBRucnmzj6UmWeGg59FgN4MAQuWBWHps0exAo/edit#slide=id.g57349aa80c_0_12

Jane

RoseFaulkner commented:
If people noticed and understood the organisation switcher then fab.

gavinwye commented:
@RoseFaulkner Yes this PR is just to put in the org switcher.

My point about not having it be applied to the Notifications page is really just that it feels odd having a global setting that's not applied to all pages. I'm wondering if that would cause more confusion. And, if there is a need for users to monitor things outside of their specific prisons.

☝️ I think we need more research on this @janereid73.

gavinwye committed:
Made the link jump directly to the section of the settings page

janereid73 commented:
I'll pop the stuff about the notifications above on to the UR plan for discussion. Thanks @gavinwye
